Ed-Fi-Exchange-OSS/EdFi-Project-Buzz
# [BIA-401] Lodash Template Vulnerability

Upgrade libraries to upgrade Lodash library. Projects updated:
- EdFi.Buzz.Api
- EdFi.Buzz.Database
- EdFi.Buzz.Etl
- EdFi.Buzz.UI

Recommended steps to test this ticket:
- Run EdFi.Buzz.Database. It should create Buzz Database.
- Run EdFi.Buzz.Etl. It should populate Buzz Database with data from ODS.
- Run API and UI. You can navigate the web site and the API should return data. Also, you could test the API using insomnia.
